index.html

```diff
@@ -41,6 +41,132 @@ <h2>LET'S CONNECT</h2>
       
     </main>
   </div>
+
+    <div class="project-wrap">
```

Comment:
Hi @yersel500 , I am impressed by the great effort you have invested in this project. I looks ⭐ beautiful ⭐. However we got a few changes to make 🤭.
- [ ] I think the structure of the projects section has been interchanged and could be the other way round according to this project requirement 👇🏻, while we make the children `<div>` tags

![image](https://user-images.githubusercontent.com/4247740/148490722-70cac7e5-da92-4a7c-a451-ba1b73115f5c.png)